waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/ThriftMetastoreClient.java

```diff
@@ -217,15 +218,23 @@ public void close() {
     }
     // Transport would have got closed via client.shutdown(), so we don't need this, but
     // just in case, we make this call.
-    if (isOpen()) {
+    if (transport != null && transport.isOpen()) {
```

Comment:
the new condition is not needed

Reply:
why not to leave it as it was?

Reply:
ok, I see.. leave the other review and approve

Reply:
the client gets closed a couple of lines above, the body of the if is just concerned with the transport